### ISSUE TITLE
Add statistics overview to the entry page

### ISSUE THREAD

**Issue by amandel** [openbikesensor/obsApp#31]
There needs to be some motivation. Add something like:
* number of tracks uploaded
* tracked km
* confirmed pass events
* number of active users
* ..

For the last 
* month
* year
* ever

**Comment by opatut** [comment]
![image](https://user-images.githubusercontent.com/235784/108269870-e6f3a300-716e-11eb-838b-1a5d4657ffe9.png)

Like so?

**Comment by amandel** [comment]
Cool - might be with higher values ;) A smaller period  like _(xx last Month)_ might also be of interests. Looking forward to see the current data.

**Comment by opatut** [comment]
Yes good idea, I will add a switch on top to choose the timeframe.

`[ This month | This year | All time ]`
